reverse_replication/launcher.go

```diff
@@ -120,6 +124,12 @@ func prechecks() error {
 	if pubSubEndpoint == "" {
 		pubSubEndpoint = fmt.Sprintf("%s-pubsub.googleapis.com:443", dataflowRegion)
 	}
+	if vpcNetwork != "" && vpcSubnetwork == "" {
+		return fmt.Errorf("if VPC network is specified, VPC subnetwork cannot be empty")
+	}
+	if vpcNetwork == "" && vpcSubnetwork != "" {
```

Comment:
We should also add a check which validates that the `subnetwork` provided in the input has the same region as the `dataflowRegion` parameter and fail fast.

Reply:
Will add in next iteration